markets/aave/reservesConfigs.ts

```diff
@@ -138,6 +138,23 @@ export const strategyBAT: IReserveParams = {
   reserveFactor: '2000'
 };
 
+export const strategyCVX: IReserveParams = {
+  optimalUtilizationRate: new BigNumber(0.45).multipliedBy(oneRay).toFixed(),
+  baseVariableBorrowRate: new BigNumber(0).multipliedBy(oneRay).toFixed(),
+  variableRateSlope1: new BigNumber(0.07).multipliedBy(oneRay).toFixed(),
+  variableRateSlope2: new BigNumber(3).multipliedBy(oneRay).toFixed(),
+  stableRateSlope1: new BigNumber(0.1).multipliedBy(oneRay).toFixed(),
+  stableRateSlope2: new BigNumber(3).multipliedBy(oneRay).toFixed(),
+  baseLTVAsCollateral: '4500',
+  liquidationThreshold: '6000',
+  liquidationBonus: '10850',
+  borrowingEnabled: true,
+  stableBorrowRateEnabled: false,
+  reserveDecimals: '18',
+  aTokenImpl: eContractid.DelegationAwareAToken,
```

Comment:
CVX has no `delegate` function, so should be `eContractid.AToken`

Reply:
Nice catch just fized